tests/Microsoft.Identity.Test.Integration.netcore/HeadlessTests/UsernamePasswordIntegrationTests.NetFwk.cs

```diff
@@ -110,12 +110,14 @@ public async Task AcquireTokenFromAdfsUsernamePasswordAsync()
             LabResponse labResponse = await LabUserHelper.GetAdfsUserAsync(FederationProvider.ADFSv2019, true).ConfigureAwait(false);
 
             var user = labResponse.User;
-
+            Uri authorityUri = new Uri(Adfs2019LabConstants.Authority);
+            
             var msalPublicClient = PublicClientApplicationBuilder
                 .Create(Adfs2019LabConstants.PublicClientId)
-                .WithAdfsAuthority(Adfs2019LabConstants.Authority)
+                .WithAuthority(authorityUri)
```

Comment:
do we need to test for `WithTenantIdFromAuthority`?

Reply:
I added a unit test. I don't think we need integration test, since the protocol is the same. This is just MSAL internal logic.